assets/js/components/GoogleChart.js

```diff
@@ -255,63 +256,66 @@ export default function GoogleChart( props ) {
 	} );
 
 	return (
-		<div
-			className={ classnames(
-				'googlesitekit-chart',
-				`googlesitekit-chart--${ chartType }`,
-				className
-			) }
-			tabIndex={ -1 }
-		>
-			<Chart
-				className="googlesitekit-chart__inner"
-				chartEvents={ combinedChartEvents }
-				chartType={ chartType }
-				chartVersion="49"
-				data={ modifiedData }
-				loader={ loader }
-				height={ height }
-				getChartWrapper={ ( chartWrapper, google ) => {
-					// An issue with `react-google-charts` v4 causes the chart to
-					// render the overlay before the chart in some cases when using
-					// their own `onLoad` callback. This is a workaround to prevent
-					// that issue but still provide notice that the chart is loaded.
-					// See: https://github.com/google/site-kit-wp/issues/4945
-					if ( ! isChartLoaded ) {
-						setIsChartLoaded( true );
-					}
+		<GoogleChartErrorHandler>
```

Comment:
Sorry, this is wrapped around the whole component… did you mean wrap this around our usage of every `GoogleChart`? If we did that, so that errors in this component were caught by `GoogleChartErrorHandler`, we might mistake some of _our own errors_ with ones caused by library bugs.

I think this is the best place to put this, but maybe I'm not following. What do you mean instead of this?

Reply:
I meant applying the wrapping around the entire `GoogleChart` component before exporting, the boundary here is around the rendered part only but I get your point. Happy to keep as-is 👍